packages/react-fresh/babel.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const ReactFreshBabelPlugin = require('./src/ReactFreshBabelPlugin');
+
+// This is hacky but makes it work with both Rollup and Jest.
+module.exports = ReactFreshBabelPlugin.default || ReactFreshBabelPlugin;
```

Comment:
I thought new packages can use ES modules.

Reply:
I don't think we enabled ESM output for anything yet because of how DEV/PROD split is done.